client/src/pages/Profile/ProfileListings/ProfileListings.tsx

```diff
@@ -0,0 +1,112 @@
+import useStyles from './useStyles';
+import Navbar from '../../../components/Navbar/Navbar';
+import {
+  Box,
+  Grid,
+  Typography,
+  TextField,
+  InputAdornment,
+  Avatar,
+  Card,
+  CardActionArea,
+  CardMedia,
+  CardContent,
+} from '@material-ui/core';
+import { Search } from '@material-ui/icons';
+import { Rating } from '@material-ui/lab';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import mockProfiles from '../../../mocks/mockProfile';
+import { Link } from 'react-router-dom';
+import { v4 as uuid } from 'uuid';
+
+const ProfileListings = () => {
+  const classes = useStyles();
+
+  return (
+    <Grid container className={classes.root}>
+      <Navbar
```

Comment:
Can we please render the Navbar in a single location and make it persistent throughout the app as opposed to rendering it within every component we create.

Reply:
@rajivtitus , can u give me some suggestions on how should I do this, like directly embedding it on top of the routing file or anything as such ...